src/PART02/DFS&BFS/01.js

```diff
@@ -0,0 +1,14 @@
+const stack = [];
+
+// 삽입(5) - 삽입(2) - 삽입(3) - 삽입(7) - 삭제() - 삽입(1) - 삽입(4) - 삭제()
+stack.push(5);
+stack.push(2);
+stack.push(3);
+stack.push(7);
+stack.pop();
+stack.push(1);
+stack.push(4);
+stack.pop();
+
+console.log(stack); // [5,2,3,1]
+console.log(stack.splice(0).reverse()) // [1,3,2,5]
```

Comment:
`Array.prototype.splice` 는 배열 그 자체를 변경시키기 때문에, 
`stack.splice(0)` 을 하면 기존의 `stack`이 빈 배열이 돼버려요
15번째 행에 `console.log(stack)` 를 한번 출력해보세요 



배열을 복제할 때는 `slice` 를 쓰는게 어떨까요? 아니면 `[...stack]` 이나 `Array.from(stack)` 도 있구요 :smile_cat: 

```suggestion
console.log(stack.slice().reverse()) // [1,3,2,5]
```